src/nidcpower/metadata/functions_addon.py

```diff
@@ -418,6 +417,52 @@
             'note': 'This function is not supported on all devices.',
         },
     },
+    'FancyCreateAdvancedSequence': {
+        'codegen_method': 'python-only',
+        'returns': 'ViStatus',
+        'python_name': 'create_advanced_sequence',
+        'method_templates': [
+            { 'session_filename': 'fancy_advanced_sequence', 'documentation_filename': 'default_method', 'method_python_name_suffix': '', },
+        ],
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'vi',
+                'type': 'ViSession',
+                'documentation': {
+                    'description': 'Identifies a particular instrument session. **vi** is obtained from the niDCPower_InitExtCal or niDCPower_InitializeWithChannels function.',
+                },
+            },
+            {
+                'direction': 'in',
+                'name': 'SequenceName',
+                'type': 'ViString',
+                'documentation': {
+                    'description': 'Name of sequence.',
```

Comment:
Does this match the LabVIEW VI string?

Reply:
Please see PR description: 'Documentation and tests still remaining but looking for feedback before doing that work'